cmd/soroban-cli/Cargo.toml

```diff
@@ -26,6 +26,10 @@ name = "soroban_cli"
 path = "src/lib.rs"
 doctest = false
 
+[features]
+default = []
+opt = ["wasm-opt"]
```

Comment:
This will cause the binary releases that we package and release on GH to not have the opt feature. Could we either make `opt` the default, which allows folks to opt-out of defaults, and change the github build for release to include this option?